eclipse-theia/theia
# Implement multiroot Open Folder command

#### What it does

Fixes #10334 by adding handling for multiselection to the `Open Folder...` command.
Fixes #10366 by implementing `resolveToAbsolute`  for URI's.

#### How to test


1. Use the 'Open Folder...' command.
2. Select multiple folders.
3. Observe that a new multi-root workspace is created and opened.
4. It should work from either an existing workspace or a no-workspace state.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: Colin Grant <colin.grant@ericsson.com>
